pkg/plugins/manager/loader/loader.go

```diff
@@ -191,14 +150,12 @@ func (l *Loader) loadPlugins(ctx context.Context, class plugins.Class, pluginJSO
 		// verify module.js exists for SystemJS to load.
 		// CDN plugins can be loaded with plugin.json only, so do not warn for those.
 		if !plugin.IsRenderer() && !plugin.IsCorePlugin() {
-			module := filepath.Join(plugin.PluginDir, "module.js")
-			if exists, err := fs.Exists(module); err != nil {
-				return nil, err
-			} else if !exists && !l.pluginsCDN.PluginSupported(plugin.ID) {
-				l.log.Warn("Plugin missing module.js",
-					"pluginID", plugin.ID,
-					"warning", "Missing module.js, If you loaded this plugin from git, make sure to compile it.",
-					"path", module)
+			_, err := plugin.FS.Open("module.js")
+			if err != nil {
+				if errors.Is(err, plugins.ErrFileNotExist) {
+					l.log.Warn("Plugin missing module.js", "pluginID", plugin.ID,
+						"warning", "Missing module.js, If you loaded this plugin from git, make sure to compile it.")
+				}
```

Comment:
Is there a reason why the `!l.pluginsCDN.PluginSupported(plugin.ID)` check was removed?

I guess we will get rid of it for the next CDN phase anyways :thinking: